app/common/themes/Base.ts

```diff
@@ -0,0 +1,493 @@
+    progressBarBg: tokens.decoration,
+
+    /* Hover */
+    hover: tokens.bgTertiary,
+
+    /* Links */
+    link: tokens.primary,
+    linkHover: tokens.primary,
+
+    /* Cell Editor */
+    cellEditorPlaceholderFg: tokens.secondary,
+    cellEditorBg: tokens.bg,
+
+    /* Cursor */
+    cursor: tokens.cursor,
+    cursorInactive: tokens.cursorInactive,
+    cursorReadonly: tokens.secondary,
+
+    /* Tables */
+    tableHeaderFg: tokens.emphasis,
+    tableHeaderSelectedFg: tokens.emphasis,
+    tableHeaderBg: tokens.bgSecondary,
+    tableBodyBg: tokens.bg,
+    tableBodyBorder: tokens.decoration,
+    tableCellSummaryBg: tokens.bgTertiary,
+
+    /* Cards */
+    cardCompactRecordBg: tokens.bg,
+    cardFormLabel: tokens.secondary,
+    cardCompactLabel: tokens.secondary,
+    cardBlocksLabel: tokens.secondary,
+    cardCompactBorder: tokens.decoration,
+    cardEditingLayoutBorder: tokens.decoration,
+
+    /* Card Lists */
+    cardListFormBorder: tokens.decoration,
+    cardListBlocksBorder: tokens.decoration,
+
+    /* Selection */
+    selectionOpaqueFg: tokens.emphasis,
+
+    /* Widgets */
+    widgetBg: tokens.bg,
+    widgetBorder: tokens.decoration,
+    widgetActiveBorder: tokens.primary,
+    widgetInactiveStripesLight: tokens.bgSecondary,
+
+    /* Pinned Docs */
+    pinnedDocFooterBg: tokens.bg,
+    pinnedDocBorder: tokens.bgTertiary,
+    pinnedDocBorderHover: tokens.secondary,
+    pinnedDocEditorBg: tokens.bgTertiary,
+
+    /* Raw Data */
+    rawDataTableBorder: tokens.bgTertiary,
+    rawDataTableBorderHover: tokens.secondary,
+
+    /* Controls */
+    controlFg: tokens.primary,
+    controlPrimaryFg: tokens.white,
+    controlPrimaryBg: tokens.primary,
+    controlPrimaryHoverBg: tokens.primaryMuted,
+    controlSecondaryFg: tokens.secondary,
+    controlSecondaryDisabledFg: tokens.decoration,
+    controlSecondaryHoverFg: tokens.body,
+    controlSecondaryHoverBg: tokens.decoration,
+
+    /* Checkboxes */
+    checkboxBg: tokens.bg,
+    checkboxSelectedFg: tokens.primary,
+    checkboxDisabledBg: tokens.decoration,
+    checkboxBorder: tokens.decoration,
+
+    /* Move Docs */
+    moveDocsSelectedFg: tokens.white,
+    moveDocsSelectedBg: tokens.primary,
+    moveDocsDisabledFg: tokens.decoration,
+
+    /* Filter Bar */
+    filterBarButtonSavedFg: tokens.white,
+    filterBarButtonSavedHoverBg: tokens.decoration,
+
+    /* Icons */
+    iconDisabled: tokens.secondary,
+
+    /* Icon Buttons */
+    iconButtonFg: tokens.white,
+    iconButtonPrimaryBg: tokens.primary,
+    iconButtonSecondaryBg: tokens.decoration,
+    iconButtonSecondaryHoverBg: tokens.secondary,
+
+    /* Left Panel */
+    activePageFg: tokens.veryLight,
+    activePageBg: tokens.bgEmphasis,
+    pageOptionsFg: tokens.secondary,
+    pageOptionsHoverFg: tokens.white,
+    pageOptionsHoverBg: tokens.decoration,
+    pageOptionsSelectedHoverBg: tokens.secondary,
+    pageInitialsFg: tokens.white,
+    pageInitialsEmojiBg: tokens.emphasis,
```

Comment:
This used to be white, is this deliberate change?

Reply:
Oops, good catch :see_no_evil: